drivers/serial/uart_bflb.c

```diff
@@ -0,0 +1,639 @@
+/*
+ * Copyright (c) 2021-2024 ATL Electronics
+ * Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT bflb_bl_uart
+
+/**
+ * @brief UART driver for Bouffalo Lab MCU family
```

Comment:
I think you would need a `@file` for doxygen comments to be picked for a driver, but I may be wrong.
It might also be possible to leave it entirely empty or without the `/**` as the drivers are documented through their API and bindings, and the comments are expected to be internal? Hope I understood the model well...

Reply:
I think this is a pattern on many drivers. Usually can point out some references.

Reply:
Thank you this makes sense, I will keep this in mind and consider adding it for new drivers.
It kind of helps having a short comment to know you landed on the right file, too.